app/blueprints/api.py

```diff
@@ -38,6 +38,22 @@ def locations():
     for entry in entries:
         item = entry.to_dict()
         locations.add(item['location'].strip())
-    
-    #set() unique-ifies the list, then convert back to list so it can be jsonify'd
+
     return jsonify(list(locations))
+
+
+@api_blueprint.route('/genres', methods=['GET'])
+def genres():
+    db = api_blueprint.config['DB']
+    db_name = os.environ['DB_NAME']
+
+    entries = db.collection(db_name).get()
+
+    genres = set()
+
+    for entry in entries:
+        item = entry.to_dict()
+        for genre in item['genre']:
+            genres.add(genre.strip())
+    
+    return jsonify(list(genres))
```

Comment:
I think we can sort the response for this and the locations before handing them off to the frontend:

```suggestion
    return jsonify(sorted(list(genres)))
```